databrowser/src/locales/en.json

```diff
@@ -178,25 +193,96 @@
       "switchViews": "Switch between views",
       "viewEmbeddedConfig": "Embedded view config",
       "viewGeneratedConfig": "Generated view config",
+      "viewUserShortLabel": "Custom",
+      "viewEmbeddedShortLabel": "Default",
+      "viewGeneratedShortLabel": "Raw",
+      "showDeprecatedFields": "Show deprecated fields",
+      "viewUserConfig": "User view config",
       "tableSettings": "Table settings",
-      "showDeprecatedFields": "Show deprecated fields"
+      "alert":{
```

Comment:
Missing space between `"alert"` and `:` indicates, that prettier rules were not applied (see also general remark on PR). We should define a strategy on how to handle formatting issues.